STYLE-DE.md

```diff
@@ -161,4 +161,12 @@ deutschen Bitcoin-Szene etabliert haben. Für einige Begriffe werden folgende
 
 ### Maßeinheiten
 
-Siehe [Hauptleitfaden](STYLE.md).
\ No newline at end of file
+Siehe [Hauptleitfaden](STYLE.md).
+
+### Nicht zu übersetzende Begriffe und Ausnahmen
+
+| Englischer Ausdruck  | Nicht übersetzen, wenn ...                                      | Beispiel/Anmerkung                                 |
+|----------------------|-----------------------------------------------------------------|----------------------------------------------------|
+| Full Node /Full Nodes| immer                                                           | Nicht als "Vollknoten" übersetzen                  |
```

Comment:
Vielleicht sollte in Zeile 75 noch angemerkt werden, dass Node im Fall von Full Node nicht übersetzt wird.